mockup-contributions_list.php

```diff
@@ -0,0 +1,922 @@
+<?php
```

Comment:
The only problem is people will then be rewriting everything in SCSS down the line. That said, as long as people know that writing anything inline will have to changed, then it should be fine.